src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import 'bulma/css/bulma.css';
+import Article from './components/Article';
```

Comment:
```suggestion
import { Article } from './components/Article';
```
Let's try to pass the test